javascript/jit_plotter_resize_pworld.js

```diff
@@ -0,0 +1,77 @@
+// create a Task event loop detecting bpatcher resizing
```

Comment:
I would rename this file `fluid.jit.plotter.labels2colors.js` to keep it namespaced and consistent.

Reply:
you mean `fluid.jit.plotter.resize.pworld.js`